src/main/java/com/fasterxml/jackson/databind/JsonNode.java

```diff
@@ -770,6 +775,14 @@ public boolean hasNonNull(int index) {
         return ClassUtil.emptyIterator();
     }
 
+    /**
+     * @return Set that can be used to traverse all key/value pairs for
+     *   object nodes; empty set (no contents) for other types
+     */
+    public Set<Map.Entry<String, JsonNode>> fields() {
```

Comment:
I think this should be `fieldSet`, as there is existing `fields` method (that returns `Iterator`)?